auto-reconfiguration.sh

```diff
@@ -9,6 +9,7 @@ VERSION=$(cat auto-reconfiguration-archives/version)
 INDEX_PATH="/auto-reconfiguration/index.yml"
 UPLOAD_PATH="/auto-reconfiguration/auto-reconfiguration-$VERSION.jar"
 
+rm -f auto-reconfiguration-archives/auto-reconfiguration-*sources.jar
```

Comment:
Yeah, _might_ come in handy if we ever have to look for a bug in a release. I hope though this will never happen. :)

Another reason is not to have unnecessary diffs against upstream; we may very well want/need to merge at some point. Then we need at least to understand _why_ this was changed.